Cargo.toml

```diff
@@ -31,4 +31,8 @@ path = "pack/src/main.rs"
 
 [[bin]]
 name = "wasm-build"
-path = "build/src/main.rs"
\ No newline at end of file
+path = "build/src/main.rs"
+
+[[bin]]
+name = "indeterminism-check"
```

Comment:
"nondeterminism" is more IT word for this (indeterminism is kinda philosophical)

also maybe just avoid negative connotation (check could be just `is_deterministic` and crate/module is just `determinism`)